pkgcore/merge/triggers.py

```diff
@@ -333,6 +334,10 @@ def _mk_ld_so_conf(self, fp):
             compatibility.raise_from(errors.BlockModification(self, e))
 
     def trigger(self, engine):
+        # ldconfig is only meaningful in GNU/Linux
+        if platform.system() != 'Linux':
```

Comment:
(this is what Portage does but it is incompatible with Jython which always returns `Java` x_x)